client_plugin/utils/fs/fs_windows.go

```diff
@@ -32,12 +32,7 @@ const (
 	FstypeDefault = ntfs
 
 	// maxDiskAttachWaitSec is the max time to wait for a disk to be attached.
-	// TODO: Reduce disk attach wait time once parallel disk identification is
-	// implemented. Currently, fs.getDiskNum(..) blocks during parallel execution
-	// due to synchronized access to ps.Exec(..). Therefore, parallel volume
-	// creation in e2e tests block in fs.DevAttachWait(..) for a while and so we
-	// allow a long delay here.
-	maxDiskAttachWaitSec = 120 * time.Second
+	maxDiskAttachWaitSec = 20 * time.Second
```

Comment:
Should also mention why its needed, even 20s seems like why its there? If each volume op is going to be in parallel there should be no wait time needed.

Reply:
It take 20-25 seconds to format `ntfs` type hence the wait time is required. I am attaching the volume creation log as shown below; please have a look at it. 

Setup detail: Volume creation from windows VM created on VMFS backed by ESX 6.0 

Command invocation:
```
PS C:\Program Files\Docker> date ; docker volume create -d vsphere Vol10 ; date                                             
Thu Nov 16 20:08:44 Coordinated Universal Time 2017                                                                             
vol10                                                                                                                       
Thu Nov 16 20:09:05 Coordinated Universal Time 2017 
```

vDVS log:
```
2017-11-16 20:08:45.1697236 +0000 GMT [ERROR] Failed to get volume meta-data error="Volume vol10 not found (file: /vmfs/volu
mes/datastore1/dockvols/11111111-1111-1111-1111-111111111111/vol10.vmdk)" name=vol10                                    
2017-11-16 20:08:46.989825 +0000 GMT [INFO] Attaching volume and creating filesystem name=vol10 fstype=ntfs                 
2017-11-16 20:08:46.989825 +0000 GMT [INFO] Waiting for a device change event                                                   
2017-11-16 20:08:48.4389095 +0000 GMT [INFO] Waiting for a watcher event volDev={3 224 17.0}                                    
2017-11-16 20:08:48.9668684 +0000 GMT [INFO] Watcher script executed stdout="EventTimedOut\n"                                   
2017-11-16 20:08:48.9668684 +0000 GMT [INFO] Successfully emitted event event=EventTimedOut                                     
2017-11-16 20:08:48.9668684 +0000 GMT [INFO] Waiting for a device change event                                                  
2017-11-16 20:08:48.9678642 +0000 GMT [INFO] Watcher emitted an event volDev={3 224 17.0} event=EventTimedOut                   
2017-11-16 20:08:50.7859716 +0000 GMT [INFO] Watcher script executed stdout="EventTimedOut\n"                                   
2017-11-16 20:08:50.7859716 +0000 GMT [WARNING] Couldn't emit event, continuing.. event=EventTimedOut                           
2017-11-16 20:08:50.7859716 +0000 GMT [INFO] Waiting for a device change event                                                  
2017-11-16 20:08:52.5390703 +0000 GMT [INFO] Watcher script executed stdout="EventTimedOut\n"                                   
2017-11-16 20:08:52.5390703 +0000 GMT [WARNING] Couldn't emit event, continuing.. event=EventTimedOut                           
2017-11-16 20:08:52.5390703 +0000 GMT [INFO] Waiting for a device change event                                                  
2017-11-16 20:08:53.4571233 +0000 GMT [INFO] Disk mapping script executed stdout="\n4\r\n" volDev={3 224 17.0}                  
2017-11-16 20:08:53.4571233 +0000 GMT [INFO] Successfully located disk volDev={3 224 17.0} diskNum=4                            
2017-11-16 20:08:53.4571233 +0000 GMT [INFO] Successfully mapped volDev to diskNum volDev={3 224 17.0} diskNum=4                
2017-11-16 20:08:53.4571233 +0000 GMT [INFO] Initiated watcher termination                                                      
2017-11-16 20:08:54.3011716 +0000 GMT [INFO] Watcher script executed stdout="EventTimedOut\n"                                   
2017-11-16 20:08:54.3011716 +0000 GMT [WARNING] Couldn't emit event, continuing.. event=EventTimedOut                           
2017-11-16 20:08:54.3011716 +0000 GMT [INFO] Watcher channels closed                                                            
2017-11-16 20:08:57.7903708 +0000 GMT [INFO] Disk mapping script executed volDev={3 224 17.0} stdout="\n4\r\n"                  
2017-11-16 20:08:57.7903708 +0000 GMT [INFO] Successfully located disk volDev={3 224 17.0} diskNum=4                            
2017-11-16 20:09:04.585771 +0000 GMT [INFO] Format disk script executed successfully fstype=ntfs label=vol10 volDev={3 224 1
7.0} diskNum=4 stdout="\r\nDriveLetter FileSystemLabel FileSystem DriveType HealthStatus OperationalStatus\r\n----------- ------
--------- ---------- --------- ------------ -----------------\r\n            vol10       NTFS       Fixed     Healthy      O
K               \r\n\r\n\r\n"                                                                                                   
2017-11-16 20:09:05.4068665 +0000 GMT [INFO] Volume and filesystem created fstype=ntfs name=vol10     
```